perception/lidar_obstacle_detection/baidu_cnn/onnx_bcnn/definition.yaml

```diff
@@ -0,0 +1,27 @@
+version: 1
+enable_testing: false
+network:
+  filename: ./model_files/bcnn.onnx
+  framework: ONNX
+  provenance: https://github.com/kosuke55/train_baiducnn
+  training: Nuscenes dataset v1.0-mini
+  model_license:
```

Comment:
please fill in a license.

Reply:
Sorry, this is still under discussion.

Reply:
@taichiH Do you have any update on this?

Reply:
@LiyouZhou sorry for my very late reply. please wait a few more days...

Reply:
@LiyouZhou I came to conclusion, I added a model_license.